ivy/functional/frontends/tensorflow/raw_ops.py

```diff
@@ -277,9 +277,7 @@ def Log(*, x, name="Log"):
     return ivy.log(x)
 
 
-@to_ivy_arrays_and_back
-def Log1p(*, x, name="Log1p"):
-    return ivy.log1p(x)
+Log1p = to_ivy_arrays_and_back(map_raw_ops_alias(tf_frontend.math.log1p))
```

Comment:
just found a small error here: https://github.com/unifyai/ivy/actions/runs/4724482305/jobs/8381749112?pr=14126#step:3:123
ivy/functional/backends/torch/elementwise.py:227: in log1p
    return torch.log1p(x, out=out)
E   RuntimeError: "log1p_vml_cpu" not implemented for 'Half'
seems like half is not an valid datatype for the torch backend, so I think you can try to add a unsupported dtype tag in the torch backend log1p function like explained here (https://lets-unify.ai/docs/ivy/overview/deep_dive/data_types.html#supported-and-unsupported-data-types)

Reply:
done

Reply:
on commit in revert tensorflow pr test2 ran successfully but this time after it is again having problem